### PR TITLE
Mosip 23833

### PR DESCRIPTION
**Pull request by ckm007**

### CODE REVIEW

File: db_upgrade_scripts/mosip_regprc/sql/1.1.5.5_to_1.2.0.1-B1_rollback.sql [hunk 1]
```diff
@@ -0,0 +1,8 @@
+\c mosip_regprc sysadmin
+
+ALTER TABLE regprc.registration DROP COLUMN IF EXISTS resume_timestamp;
+ALTER TABLE regprc.registration DROP COLUMN IF EXISTS default_resume_action;
```
Comment (sowmya695) [inline db_upgrade_scripts/mosip_regprc/sql/1.1.5.5_to_1.2.0.1-B1_rollback.sql:4]: @ckm007  As part of release-1.2.0.1 many other changes for same table and other tables in regproc, Why it is added only for these two columns here.These columns are not present in 1.1.5.5

**Review by sowmya695 — CHANGES_REQUESTED** [review]
Please check the one of clarification needed

File: db_upgrade_scripts/mosip_regprc/upgrade.properties [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+DB_SERVERIP=
+DB_PORT=
+SU_USER=postgres
+SU_USER_PWD=
+SYS_ADMIN_USER=
+SYS_ADMIN_PWD=
+DEFAULT_DB_NAME=postgres
+DBUSER_PWD=
+CURRENT_VERSION=
+TARGET_VERSION=
```
Comment (sowmya695) [inline db_upgrade_scripts/mosip_regprc/upgrade.properties:11]: Please update it as UPGRADE_VERSION

File: db_upgrade_scripts/mosip_regprc/upgrade.properties [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+ACTION=
```
Comment (sowmya695) [inline db_upgrade_scripts/mosip_regprc/upgrade.properties:12]: Please move this to first line as it was not working at the end

File: db_upgrade_scripts/mosip_regprc/upgrade.sh [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+echo "UPGRADE version: $UPGRADE_VERSION"
+echo "Action: $ACTION"
+
+# Terminate existing connections
+echo "Terminating active connections"
+CONN=$(PGPASSWORD=$SU_USER_PWD psql -v ON_ERROR_STOP=1 --username=$SU_USER --host=$DB_SERVERIP --port=$DB_PORT --dbname=$DEFAULT_DB_NAME -t -c "SELECT count(pg_terminate_backend(pg_stat_activity.pid)) FROM pg_stat_activity WHERE datname = '$MOSIP_DB_NAME' AND pid <> pg_backend_pid()";exit;)
+echo "Terminated connections"
+
+# Execute upgrade or rollback
+if [ "$ACTION" == "upgrade" ]; then
```
Comment (sowmya695) [inline db_upgrade_scripts/mosip_regprc/upgrade.sh:28]: Please replace "$ACTION" with $ACTION

**Review by sowmya695 — CHANGES_REQUESTED** [review]
Please fix the comments

**Review by sowmya695 — CHANGES_REQUESTED** [review]
Please fix the comments

**Review by sowmya695 — CHANGES_REQUESTED** [review]
Please fix the comments

**Review by vishwa-vyom — APPROVED** [review]
The open comments will be taken care by Sowmya in the new PR, so this PR is approved

**Review by sowmya695 — APPROVED** [review]
I will do the necessary changes

**Pull request merged**